test/MinterGateway.t.sol

```diff
@@ -1394,18 +1399,22 @@ contract MinterGatewayTests is TestUtils {
         );
 
         uint240 activeOwedM = _minterGateway.activeOwedMOf(_minter1);
-
         uint256 maxAllowedOwedM = (collateral * _mintRatio) / ONE;
+        uint256 excessOwedM = activeOwedM - maxAllowedOwedM;
 
         uint240 penalty = uint240(
-            ((((activeOwedM - maxAllowedOwedM) * (_updateCollateralInterval - 1)) / _updateCollateralInterval) *
-                _penaltyRate) / ONE
+            (((excessOwedM * (_updateCollateralInterval - 1)) / _updateCollateralInterval) * _penaltyRate) / ONE
         );
 
         assertEq(_minterGateway.getPenaltyForMissedCollateralUpdates(_minter1), 0);
 
         vm.expectEmit();
-        emit IMinterGateway.PenaltyImposed(_minter1, _minterGateway.getPrincipalAmountRoundedDown(penalty), penalty);
+        emit IMinterGateway.UndercollateralizedPenaltyImposed(
+            _minter1,
+            uint240(excessOwedM) + 1,
```

Comment:
Why do we need to round up by 1 here?

Reply:
Because the tests is doing present value math, when the contract is doing principal value math.